loomi/regulariza-diagonal-client
# Feature/notification bar

## Feature
 - NotificationDrawer
---

**What I did:**
  - Create the components:
     - NotificationDrawer
     - NotificationButton
     - NotificationContainer
     - BaseDrawer
 - Create the context:
     - DrawerContext
 - Put it all new components in DashHeader under the DrawerContext

**How to test:**

1. In your browser go to [stage](http://project.netlify.app/) or [localhost](http://localhost:3000/)
2. Check if the component/view looks just like the [mockup](https://xd.adobe.com/) in different screen sizes (width between 1024px and 1920px)
3. Test the things that you think are worth testing, even the ones that are not in this description


![image](https://user-images.githubusercontent.com/32523401/131178047-7ae75906-1be5-4c71-bca5-64fcd2d38bac.png)
![image](https://user-images.githubusercontent.com/32523401/131178072-dbac86c1-67fd-4842-9842-d71479f2e5ef.png)
